src/main/java/frc/robot/Constants.java

```diff
@@ -198,8 +198,16 @@ public static class PhotonVisionConstants {
 
   public static class IntakeConstants {
     //TODO: Assign a real CAN ID
-    public static final int kIntakeMotorID = 0;
+    public static final int kTopIntakeMotorID = 0;
+    public static final int kBottomIntakeMotorID = 1;
     public static final int kIntakeCurrentLimit = 40;
+    public static final double kShaftDiameter = 0.0127;
```

Comment:
By shaft diameter, do you mean intake wheel diameter? That's `Units.inchesToMeters(1.0)` for the bottom intake roller, and `Units.inchesToMeters(2.0)` for the top intake roller.

Reply:
I made new commands that replace ShaftDiameter. I don't know where we got the number for it but the intake wheel diameter number is the correct number for the calculation ShaftDiameter was going to perform.